starfish/plot/intensity_table.py

```diff
@@ -0,0 +1,182 @@
+import warnings
```

Comment:
i find it confusing to have two intensity_table.pys in the project, even if one is in starfish.plot and the other is in starfish. is there a better name for this file? can we move the function into plots.spots.py?